.github/workflows/ci.yml

```diff
@@ -665,12 +665,24 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.TRINO_AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.TRINO_AWS_SECRET_ACCESS_KEY }}
           AWS_REGION: ${{ vars.TRINO_AWS_REGION }}
-          S3_BUCKET: ${{ vars.TRINO_S3_S3FS_TESTS_BUCKET }}
         if: >-
           contains(matrix.modules, 'trino-filesystem-s3') && contains(matrix.profile, 'cloud-tests') &&
           (env.CI_SKIP_SECRETS_PRESENCE_CHECKS != '' || env.AWS_ACCESS_KEY_ID != '' || env.AWS_SECRET_ACCESS_KEY != '')
         run: |
+          # Create an empty S3 bucket for S3 filesystem cloud tests and add the bucket name to GitHub environment variables
+          .github/bin/s3/setup-empty-s3-bucket.sh
+          EMPTY_S3_BUCKET=$(cat .github/bin/s3/.bucket-identifier)
+          export EMPTY_S3_BUCKET
```

Comment:
You can make this
```
export EMPTY_S3_BUCKET=$(cat .github/bin/s3/.bucket-identifier)

Reply:
Jan pointed out in a previous comment that having them on the same line could mask any errors returned by `cat` https://github.com/trinodb/trino/pull/19060#discussion_r1333461788